package-res/resources/web/dojo/pentaho/common/overrides/dojo/_base/kernel.js

```diff
@@ -10,6 +12,7 @@ define(["../has", "./config", "require", "module"], function(has, config, requir
 
 		// create dojo, dijit, and dojox
 		// FIXME: in 2.0 remove dijit, dojox being created by dojo
+		global = (function () { return this; })(),
 		dijit = {},
```

Comment:
This version of require.js changed the value of `this` when calling the module definition function, from the global object to the module's exports object, so uses of `this` directly in the definition function broke. But not uses of this in other functions ;-) as is the case of the determination of the `global` variable.